test/integration/an_existing_user_can_check_out_to_place_an_order_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class AnExistingUserCanCheckOutToPlaceAnOrderTest < ActionDispatch::IntegrationTest
+  include CategoryItemsSetup
+  test "a visitor must log in before checking out" do
+    create_categories_and_items
+    add_items_to_cart
+
+    visit '/cart'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.